src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -193,7 +193,8 @@ public void initFlowFrameworkIndexIfAbsent(FlowFrameworkIndex index, ActionListe
                     logger.error(errorMessage, e);
                     internalListener.onFailure(new FlowFrameworkException(errorMessage, ExceptionsHelper.status(e)));
                 });
-                CreateIndexRequest request = new CreateIndexRequest(indexName).mapping(mapping).settings(indexSettings);
+                CreateIndexRequest request = new CreateIndexRequest(indexName).mapping("{\"_doc\":" + mapping + "}")
```

Comment:
Would we need it [here](https://github.com/opensearch-project/flow-framework/blob/034dff7220a300187b2cf262cacb029faf13ebcd/src/main/java/org/opensearch/flowframework/workflow/CreateIndexStep.java#L102) as well?

Reply:
> Would we need it [here](https://github.com/opensearch-project/flow-framework/blob/034dff7220a300187b2cf262cacb029faf13ebcd/src/main/java/org/opensearch/flowframework/workflow/CreateIndexStep.java#L102) as well?

Good question.  

I don't think so, that method says nothing about `_doc`: https://github.com/opensearch-project/OpenSearch/blob/157d27700157f3e24ef8b150b542e78d788bddab/server/src/main/java/org/opensearch/action/admin/indices/create/CreateIndexRequest.java#L512